open-metadata-implementation/access-services/cognos/cognos-api/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<parent>
+		<artifactId>cognos</artifactId>
+		<groupId>org.odpi.egeria</groupId>
+		<version>1.8-SNAPSHOT</version>
+	</parent>
+
+	<name>Cognos OMAS API</name>
+	<description>
+        Shared beans for the Cognos Open Metadata Access Service (OMAS).
```

Comment:
I think Mandy mentioned this, but the intent would be to have more generalization with the intent (albeit difficult on a sample of 1) to support different tools in the same class - so I wouldn't expect the omas to be cognos specific

Reply:
plugin is renamed to Analytics Modeling as discussed.